src/icons/icon-success-vivo.tsx

```diff
@@ -5,16 +5,15 @@ import {useIsInverseVariant} from '../theme-variant-context';
 import {vars} from '../skins/skin-contract.css';
 
 type Props = {
-    role?: string;
     size?: number | string;
 };
 
-const IconSuccessVivo: React.FC<Props> = ({role = 'presentation', size = 64}) => {
+const IconSuccessVivo: React.FC<Props> = ({size = 48}) => {
     const isInverse = useIsInverseVariant();
     const {platformOverrides} = useTheme();
 
     return (
-        <svg role={role} width={size} height={size} viewBox="0 0 64 64" style={{marginLeft: -17}}>
+        <svg width={size} height={size} viewBox="0 0 64 64" style={{marginLeft: -10}}>
             <g
                 transform="translate(17, 10)"
```

Comment:
This was a weird hack to avoid overflowing content from being cropped by the container. I removed both the `transform translate` and the `marginLeft`, and used `overflow: visible` in the container.